Doberman/AlarmNode.py

```diff
@@ -79,9 +85,7 @@ def setup(self, **kwargs):
         self.sensor_config_needed += ['alarm_recurrence']
 
     def process(self, package):
-        # the 2 is a fudge factor
-        dt_max = (2 + self.config['alarm_recurrence']) * self.config['readout_interval']
-        if (dt := ((now := time.time()) - package['time'])) > dt_max:
+        if (dt := ((now := time.time()) - package['time'])) > self.max_reading_delay:
```

Comment:
When your max_reading_delay is lower than your readout interval this triggers all the time, doesn't it? Should this be `self.config['readout_interval'] + self.max_reading_delay`

Reply:
You're right, good point